notebooks/Data_Attribute_Recommendation_Regression_Model_Template.ipynb

```diff
@@ -0,0 +1,1631 @@
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5aZ-dcKrq-z7"
+      },
+      "source": [
+        "*Note: if you are doing several runs of this notebook on a trial account, you may see errors stating 'The resource can no longer be used. Usage limit has been reached'. It can be beneficial to [clean up the service instance](#Cleaning-up-a-service-instance) to free up limited trial resources acquired by an earlier run of the notebook. [Some limits](https://help.sap.com/viewer/105bcfd88921418e8c29b24a7a402ec3/SHIP/en-US/c03b561eea1744c9b9892b416037b99a.html) cannot be reset this way.*"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZpQtE58eq-z7"
+      },
+      "source": [
+        "## Table of Contents\n",
+        "\n",
+        "* [Exercise 01.1](#Exercise-01.1) - Installing the SDK and preparing the service key\n",
+        "    * [Creating a service instance and key on BTP Trial](#Creating-a-service-instance-and-key)\n",
+        "    * [Installing the SDK](#Installing-the-SDK)\n",
+        "    * [Loading the service key into your Jupyter Notebook](#Loading-the-service-key-into-your-Jupyter-Notebook)\n",
+        "* [Exercise 01.2](#Exercise-01.2) - Uploading the data\n",
+        "* [Exercise 01.3](#Exercise-01.3) - Training the model\n",
+        "* [Exercise 01.4](#Exercise-01.4) - Deploying the Model and predicting labels\n",
+        "* [Resources](#Resources) - Additional reading\n",
+        "* [Cleaning up a service instance](#Cleaning-up-a-service-instance) - Clean up all resources on the service instance\n",
+        "* [Optional Exercises](#Optional-Exercises) - Optional exercises"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "g4hrCmUhq-z8"
+      },
+      "source": [
+        "## Requirements\n",
+        "\n",
+        "\n",
+        "See the [README in the Github repository for this workshop](https://github.com/SAP-samples/teched2020-INT260/blob/master/exercises/ex1-DAR/README.md)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fIE4U09vq-z8"
+      },
+      "source": [
+        "# Exercise 01.1\n",
+        "\n",
+        "*Back to [table of contents](#Table-of-Contents)*\n",
+        "\n",
+        "In exercise 01.1, we will install the SDK and prepare the service key."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Lg-q4Sqfq-z9"
+      },
+      "source": [
+        "## Creating a service instance and key on BTP Trial"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CPIoP_fUq-z9"
+      },
+      "source": [
+        "Please log in to your trial account: https://cockpit.eu10.hana.ondemand.com/trial/\n",
+        "\n",
+        "In the your global account screen, go to the \"Boosters\" tab:\n",
+        "\n",
+        "![trial_booster.png](attachment:trial_booster.png)\n",
```

Comment:
I think the attachments got lost during the cleanup. The screenshots were present in the initial commit.

Reply:
I've now included the screenshots as file paths. I'm unable to add images as attachments as you did. We need to see how you did that and replicate in all the notebooks.